lib/base-error.js

```diff
@@ -0,0 +1,32 @@
+var _ = require('underscore');
```

Comment:
Probably not in scope for now, but I'd been keen to know how far we are from `s/underscore/lodash`.